16_woman_cancer.do

```diff
@@ -11,8 +11,11 @@ if _rc==0 {
 	replace w_papsmear=. if s714dd==9 | s714ee==9
 }
 
-
 capture confirm variable s1017 s1020 
 if _rc==0 {
     replace w_mammogram=. if s1017==. | s1017==9 | s1020==9
 }
+
+// I'm wodering where do you find these codes? 

```

Comment:
It's from the recode VI adeptfile. So far I didn't see any different code on this one in the country specific case.